site/content/docs/5.2/utilities/background.md

```diff
@@ -12,6 +12,17 @@ toc: true
 
 Similar to the contextual text color classes, set the background of an element to any contextual class. Background utilities **do set `color`** to ensure contrasts.
 
+{{< callout info >}}
```

Comment:
I think I prefer this callout after the example

Reply:
Yep it's better. Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/7659e56f784d0184be9143646169115da32033c2